internal/ask/form/form_test.go

```diff
@@ -0,0 +1,115 @@
+package form_test
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/ardanlabs/kit/cfg"
+	"github.com/ardanlabs/kit/db"
+	"github.com/ardanlabs/kit/db/mongo"
+	"github.com/ardanlabs/kit/tests"
+	"github.com/coralproject/shelf/internal/ask/form"
+	"github.com/coralproject/shelf/internal/ask/form/formfix"
+)
+
+var dbSession *db.DB
+
+// prefix is what we are looking to delete after the test.
+const prefix = "FTEST_"
+
+func TestMain(m *testing.M) {
+	// Initialize the configuration and logging systems. Plus anything
+	// else the web app layer needs.
+	tests.Init("XENIA")
```

Comment:
shouldnt' the domain here be "TEST" ? In that case we do specific strings for the tests DB. And Jenkins can setup this different.